tests/alternates.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(plugin)]
+#![feature(plugin,const_fn)]
```

Comment:
Can't say for sure, it was caused by:

https://github.com/reem/stainless/pull/28/files#diff-6755e3579c3e8158c7d672a1ddebb24dR87

and that change was required because the new function also needs that argument.